packages/mux-player/src/template.ts

```diff
@@ -41,8 +43,8 @@ const getHotKeys = (props: MuxTemplateProps) => {
 export const content = (props: MuxTemplateProps) => html`
   <media-theme
     template="${props.themeTemplate ?? muxTemplate.content.children[0]}"
-    stream-type="${isLiveOrDVR(props) ? 'live' : 'on-demand'}"
-    target-live-window="${isDVR(props) ? 1 : false}"
+    default-stream-type="${props.defaultStreamType}"
+    class="${props.secondaryColor ? 'two-tone' : false}"
```

Comment:
even though this was broken in the old impl, should fix the "seek to live on play" for "simulated live" cases.

Reply:
Resolved for "DVR". Will treat as followup for "simulated live".